File: szxy-funplay-parent/funplay-sellergoods-interface/src/main/java/com/funplay/sellergoods/service/SpecificationService.java
package com.funplay.sellergoods.service;

import com.funplay.pojo.Specification;
import com.funplay.pojogroup.SpecificationGroup;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface SpecificationService {

    public List<Specification> findAll();


    public PageResult findPage(Specification specification, int pageNum, int pageSize);

    public void add(SpecificationGroup specificationGroup);


    public SpecificationGroup findOne(Long id);

    public void update(SpecificationGroup specificationGroup);

    public void delete(Long[] ids);


    //查询规格列表加入模板的新增中
    public List<Map> selectOptionList();

}

File: szxy-funplay-parent/funplay-manager-web/src/main/webapp/js/service/specificationService.js
//服务层
app.service('specificationService',function($http){

	//查询实体
	this.findOne=function(id){
		return $http.get('../specification/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../specification/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../specification/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../specification/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../specification/search.do?page='+page+"&rows="+rows, searchEntity);
	}

	//规格下拉列表
	this.selectOptionList=function () {
		return $http.get('../specification/selectOptionList.do');
    }
});
